Pidgin/Parser.Number.cs

```diff
@@ -143,5 +145,54 @@ private static long GetLetterOrDigitValueLong(char c)
         }
         private static long GetUpperLetterOffsetLong(char c) => c - 'A';
         private static long GetLowerLetterOffsetLong(char c) => c - 'a';
+		
+		/// <summary>
+        /// A parser which parses a floating point number with an optional sign.
+        /// The resulting <c>double</c> is not checked for overflow.
+        /// </summary>
+        /// <returns>A parser which parses a floating point number with an optional sign</returns>
+        public static Parser<char, double> Float { get; } = RealNum();
```

Comment:
`Float` is a confusing name for a parser which returns a `double` (and it uses up the name `Float` for if we wanted to add one which does return `float`).

Reply:
You are correct, Float obviously indicates a single precision floating point number.  We could call it 'DoubleNum' or 'FloatDouble' or anything else that you would like.  I defer this choice to you as the owner of the project.

Reply:
I'm still drawing blanks on the name for this. Maybe `Fractional`?

Reply:
hehe i like the idea that users will have to figure out how to type é into their editors